integration-tests/src/lib.rs

```diff
@@ -719,6 +728,16 @@ pub fn build_test_config_from_parts_and_names(
         );
     }
 
+    if *RUNS_IN_CONTAINER {
+        extend_config(
+            &mut config,
+            toml! {
+                [text_extractor]
+                url = "http://tika:9998/"
+            },
+        );
+    }
```

Comment:
This is probably what is causing the problems on CI.

For tests which do not use `with_text_extractor_options` this lead to there being a:

```
                [text_extractor]
                url = "http://tika:9998/"
```

config without any other fields

which fails to deserialize due to missing the enum tag

I'm not sure why deserialization errors in the config aren't displayed properly anymore (or maybe never where :thinking: ).

Reply:
Was that, thanks.